guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md

```diff
@@ -125,18 +125,18 @@ class CustomerAgeContextPlugin
 
 The `subject->setValue` argument specifies the value for newcomer context and is used to guarantee parity during cache key generation for newcomers and users who already received the `X-Magento-Vary` cookie.
 
-For another example of a context class, see [Magento/Framework/App/Http/Context]({{ site.mage2000url }}lib/internal/Magento/Framework/App/Http/Context.php){:target="_blank"}.
+For another example of a context class, see [Magento/Framework/App/Http/Context]({{ site.mage2300url }}lib/internal/Magento/Framework/App/Http/Context.php){:target="_blank"}.
```

Comment:
```suggestion
For another example of a context class, see [Magento/Framework/App/Http/Context]({{ site.mage2100url }}lib/internal/Magento/Framework/App/Http/Context.php){:target="_blank"}.
```

Reply:
Hi @erikmarr,
So the is the change the variable to mage2100url instead? Then the 2.3 pages will get the 2.1 repository still, right?

Reply:
Yeah linking to 2.0 needs to be fixed and while 2.1 is not ideal either, it's the compromise for now.